src/guide/jbs-jdk-bug-system.md

```diff
@@ -345,7 +345,17 @@ This table contains some frequently used JBS labels and their meaning. Please he
   <tr>
     <td class="dictionary">[*(Rel)*[-na]{.jbs-label}]{#rel-na}</td>
     <td class="dictionary">
-      The [Affects Version/s]{.jbs-field} field is used to indicate the releases where an issue has been seen - it's implied that the issue is also applicable to newer releases. Where this isn't the case - for instance a bug in code that was removed in *(Rel)* - then use the *(Rel)-na* to indicate this. Note that there should only be **one** *(Rel)*[-na]{.jbs-label} label on any JBS issue.
+      Labels of the form [11-na]{.jbs-label} or [21-na]{.jbs-label} should be used when a bug is not applicable to a **more recent** release family.
+
+      The [Affects Version/s]{.jbs-field} field is used to indicate the releases where an issue has been seen - it's implied that the issue is also applicable to newer releases. Where this isn't the case - for instance a bug in code that was removed in *(Rel)* - then use *(Rel)*[-na]{.jbs-label} to indicate this. Note that there should only be **one** *(Rel)*[-na]{.jbs-label} label on any JBS issue.
+
+      Don't use the label to indicate that a bug isn't relevant to an earlier release. If a bug is not present in an earlier release you can use [Introduced in Version]{.jbs-field} to indicate where the bug was introduced. If a bug is present in an earlier release, but shouldn't be fixed there, use [*(Rel)*[-wnf]{.jbs-label}]{#rel-wnf}.
```

Comment:
But from a backporting perspective I, for example, would want to know if the issue is present in 8u, and I can easily test that. But I don't know if the issue in present in 9 or 10 and I would not want to waste time testing those versions just so I can set the "Introduced In" field. It makes more sense to me, to use `8-na` to indicate that JDK 8 has been checked and has no issue. Alternatively, the guide should be directing me to perhaps create an explicit backport record for 8u and close it as "Not an Issue".

Reply:
The intention is to have an upper boundary on the need to fix an issue - the X-na label, and a lower boundary - the X-wnf label. A label that is used to spot-indicate specific releases without saying anything about the before or after is much less useful when trying to automate things. For a human looking at a single bug what you say makes sense, but for a tool it would be impossible to determine if 8-na means just 8 or 8 and later.

The labels are slightly badly named which is why this discussion exists. But this is what we currently have. To change the names of the labels is a much larger discussion.